data/autoyast_hanaperf/sle15.xml.ep

```diff
@@ -35,10 +35,10 @@
         <media_url><![CDATA[http://dist.suse.de/install/SLP/SLE-<%= $get_var->('VERSION') %>-Module-Basesystem-GM/<%= $get_var->('ARCH') %>/DVD1/]]></media_url>
         <product_dir>/</product_dir>
         % } else {
-        <media_url><![CDATA[http://dist.suse.de/install/SLP/SLE-<%= $get_var->('VERSION') %>-Full-GM/<%= $get_var->('ARCH') %>/DVD1/]]></media_url>
+	<media_url><![CDATA[<%= $get_var->('MIRROR_HTTP') %>]]></media_url>
```

Comment:
is it new or existing setting?

Reply:
Seems like it not only has the hostname but the full URL also with the folder name. Am I correctly understand it?

Reply:
Yes, it is an existing setting, looks like 
`MIRROR_HTTP  http://openqa.suse.de/assets/repo/fixed/SLE-15-SP5-Full-x86_64-GM-Media1/`